bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/provider/AbstractResourceBundleProvider.java

```diff
@@ -395,7 +387,8 @@ protected Bundle getBundle(String uid) {
     }
 
     protected List<ConfigDescriptionParameter> getLocalizedConfigurationDescription(TranslationProvider i18nProvider,
-            List<ConfigDescriptionParameter> config, Bundle bundle, String uid, String prefix, Locale locale) {
+            @Nullable List<ConfigDescriptionParameter> config, @Nullable Bundle bundle, String uid, String prefix,
+            Locale locale) {
```

Comment:
```suggestion
            @Nullable Locale locale) {
```

Reply:
Many methods got annotated this way but I seemed to have missed that one. :+1: